sdk/canton/community/app-base/src/main/scala/com/digitalasset/canton/config/CantonConfig.scala

```diff
@@ -82,7 +82,7 @@ import com.digitalasset.canton.pureconfigutils.HttpServerConfig
 import com.digitalasset.canton.pureconfigutils.SharedConfigReaders.catchConvertError
 import com.digitalasset.canton.sequencing.authentication.AuthenticationTokenManagerConfig
 import com.digitalasset.canton.sequencing.client.SequencerClientConfig
-import com.digitalasset.canton.telemetry.HistogramDefinition
+import com.daml.metrics.HistogramDefinition
```

Comment:
Is there another PR planned after this one, when the code is physically moved to the `observability` project?

Reply:
Mentioned, because this change will be overwritten next day with the fresh code drop